### ISSUE TITLE
Title and role attributes are missing for functional control  in Widget Builder and Admin Screen

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#786]
**Describe the bug**
Title and role attributes are missing for functional control  in Widget Builder screen.

**To Reproduce**
1. Go to Widget Builder (http://localhost:9992/cm/app/?view=widgetbuilder&site=test&path=%2FSites%2Ftest%2F&pathType=page)
2. Click on Ceate New Widget button.
3. Mouse over on  the check box given near to "Is Responsive" label.
4. No title/label displayed.
5. In Admin screen, role and title also missing for pop-ups, which appears when user click on + icons, given against workflow and assigned sections, under workflow tab.

**Expected behavior**
Title/Label should be displayed on each component of the screen.
